modules/storage-api/src/test/java/org/apache/ignite/internal/storage/AbstractMvTableStorageTest.java

```diff
@@ -118,14 +118,18 @@ void testPartitionIndependence() throws Exception {
 
         UUID txId = UUID.randomUUID();
 
-        RowId rowId0 = partitionStorage0.runConsistently(() -> partitionStorage0.insert(testData0, txId));
+        RowId rowId0 = new RowId(1);
```

Comment:
paririonId is actually 42 here. Please add variable storage0PartitionId to use both getOrCreateMvPartition() and rowId0.

Reply:
Done.